google/cloud/forseti/common/opencensus/tracing.py

```diff
@@ -0,0 +1,347 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Forseti OpenCensus gRPC tracing setup."""
+
+import functools
+import inspect
+import logging
+
+from google.cloud.forseti.common.util import logger
+
+LOGGER = logger.get_logger(__name__)
+# set debug level for opencensus
+logger.get_logger('opencensus').setLevel(logging.DEBUG)
```

Comment:
I don't see where this `logger` (lowercase) is being used?

Reply:
This is setting all the`opencensus` logs to debug level based on `get_logger()` implementation.

Reply:
@red2k18 was used for debug only to get more detail on traces. Can we removed now.